hail/python/hailtop/aiocloud/aioazure/fs.py

```diff
@@ -500,7 +552,7 @@ async def close(self) -> None:
             self._credential = None
 
         if self._blob_service_clients:
-            await asyncio.wait([client.close() for client in self._blob_service_clients.values()])
+            await asyncio.wait([asyncio.create_task(client.close()) for client in self._blob_service_clients.values()])
```

Comment:
How come you've wrapped this in a task?

Reply:
asyncio.wait needs a task - looks like Jackie already made this change upstream